app/routes/games/helpers.js

```diff
@@ -51,7 +51,10 @@ function renderPage(req, res, template, populate=null)
 			{
         var game = Game.getBySlug(req.params.slug, done);
         if (populate){
-          game.populate(populate);
+          game.populate({
```

Comment:
Does this code not need to be ran before calling done? 
Would it be worth making sure that this runs first before going to the next function?

Reply:
Okay, we figured this out. The variable is named little incorrectly here: `game` is actually a "Query" instance. So, the order of operations actually looks something like this:

```javascript
// start a query that will invoke done when it finishes. Put it in a variable called "game"
var game = Game.getBySlug('slug', done);

// add more clauses, joins, etc. on to the query. These clauses will have been applied BEFORE
// the done callback is called
game.populate({
  // some options.
});
```

Reply:
On that note, I think it's fine like it is, but _could_ use a variable rename.